exonum-java-binding-core/src/test/java/com/exonum/binding/crypto/Ed25519CryptoFunctionTest.java

```diff
@@ -36,18 +40,20 @@
 
   @Test
   public void generateKeyPairWithSeed() {
-    byte[] seed = new byte[64];
+    byte[] seed = new byte[SEED_BYTES];
 
     KeyPair keyPair = CRYPTO_FUNCTION.generateKeyPair(seed);
     assertNotNull(keyPair);
+    assertTrue(hasLength(keyPair.getPrivateKey().toBytesNoCopy(), PRIVATE_KEY_BYTES));
```

Comment:
assertThat(key.size(), equalTo(expectedSize))? — It would result in a more detailed error message in case the assertions fails.